addons/im_livechat/models/im_livechat_channel.py

```diff
@@ -92,6 +92,16 @@ def action_quit(self):
         self.ensure_one()
         return self.write({'user_ids': [(3, self._uid)]})
 
+    def action_view_channel_ratings(self):
+        self.ensure_one()
+        action = self.env['ir.actions.act_window']._for_xml_id('im_livechat.rating_rating_action_view_livechat_rating')
+        action['display_name'] = _('Ratings for %s') % self.name
```

Comment:
use named placeholder for translation

Reply:
Are you suggesting something like `action['display_name'] = _('Ratings for %(channel_name)s', channel_name=self.name)` ?

Reply:
Yes, this helps translator to know what kind of information they receive.